dbms/src/Storages/System/StorageSystemDictionaries.cpp

```diff
@@ -36,10 +36,12 @@ NamesAndTypesList StorageSystemDictionaries::getNamesAndTypes()
         {"element_count", std::make_shared<DataTypeUInt64>()},
         {"load_factor", std::make_shared<DataTypeFloat64>()},
         {"source", std::make_shared<DataTypeString>()},
+        {"dictionary_lifetime_min", std::make_shared<DataTypeUInt64>()},
+        {"dictionary_lifetime_max", std::make_shared<DataTypeUInt64>()},
```

Comment:
The prefix "dictionary_" seems useless because the table `system.dictionaries` shows information about dictionaries only.